application/src/main/scala/com/azavea/franklin/database/StacItemDao.scala

```diff
@@ -59,12 +59,11 @@ object StacItemDao extends Dao[StacItem] {
       .withUniqueGeneratedKeys[StacItem]("item")
   }
 
-  def getCollectionItem(collectionId: String, itemId: String): ConnectionIO[Option[StacItem]] = {
+  def getCollectionItem(collectionId: String, itemId: String): ConnectionIO[Option[StacItem]] =
```

Comment:
lol this was just a joke sorry. We should probably just add the scalafix rule to always include them, since adding them to add intermediate assignments / logging / etc. is inconvenient and I apparently can't be trusted not to add facetious :rage: emojis to them